client/files.go

```diff
@@ -606,33 +608,42 @@ func (client *Client) Pull(opts *PullOptions) error {
 	}
 	defer responsePart.Close()
 	if responsePart.FormName() != "response" {
-		return fmt.Errorf(`expected second field name to be "response", got %q`, responsePart.FormName())
+		return fmt.Errorf("expected second field name to be \"response\", got %q", responsePart.FormName())
```

Comment:
Again, let's avoid unnecessary stylistic changes in this PR. In this case, the backticks were there for a reason, as they avoid the escaped quotes inside the string.